python/tank/authentication/user_impl.py

```diff
@@ -206,7 +205,9 @@ def __init__(self, host, login, session_token, http_proxy, password=None, cookie
         self._login = login
         self._session_token = session_token
         self._cookies = cookies
-        self._saml_expiration = saml_expiration
+
+        if self._cookies:
+            logger.debug("Cookie jar: %s" % self._cookies)
```

Comment:
feels like this log message needs a bit more context to be truly helpful for tech support.